server.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const app = express();
```

Comment:
Consider adding error handling for your server. This can be done by adding a middleware function after all your routes that catches any errors and sends a response with the error message. [important]